nnvm/python/nnvm/frontend/mxnet.py

```diff
@@ -188,12 +188,16 @@ def _reshape(inputs, attrs):
     return _get_nnvm_op(op_name)(*inputs, **new_attrs)
 
 def _split(inputs, attrs):
-    if _parse_bool_str(attrs, 'squeeze_axis'):
-        _raise_not_supported('squeeze_axis', 'split')
     op_name, new_attrs = 'split', {}
     new_attrs['indices_or_sections'] = _required_attr(attrs, 'num_outputs')
-    new_attrs['axis'] = attrs.get('axis', 1)
-    return _get_nnvm_op(op_name)(*inputs, **new_attrs)
+    axis = attrs.get('axis', 1)
+    new_attrs['axis'] = axis
+    outputs = _get_nnvm_op(op_name)(*inputs, **new_attrs)
+    if _parse_bool_str(attrs, 'squeeze_axis'):
+        squeeze_attrs = {'axis': axis}
+        for o in outputs:
+            o = _get_nnvm_op('squeeze')(o, **squeeze_attrs)
```

Comment:
are you sure this can be replaced in-place?

Reply:
Oh, it was my mistake. This cannot be replaced in-place. I'll fix it, but I need to find how to update the split symbol with squeeze. Please let me know if you have any good ideas.